code/modules/mob/living/carbon/human/species/emote-synth_k9.dm

```diff
@@ -0,0 +1,31 @@
+/datum/emote/living/carbon/human/synthetic/synth_k9/New()  //K9's are blacklisted from human emotes on emote.dm, we need to not block the new emotes below
+	. = ..()
+	species_type_blacklist_typecache -= /datum/species/synthetic/synth_k9
```

Comment:
Will null this out, originally I figured it would be future proofing but given that this runs once and isn't stored (I was unaware) it's probably just best to have it set to NULL.